js/selectors/activity-feedback-selectors.js

```diff
@@ -57,18 +59,15 @@ const activityFeedbackFor = (activity, student, feedbacks) => {
 };
 
 const feedbackIsMarkedComplete = (fb) => {
-  const feedback = fb.get("feedbacks") && fb.get("feedbacks").first();
-  return feedback && feedback.get("hasBeenReviewed");
+  return fb && fb.get("hasBeenReviewed");
 };
 
-const hasLearner = (feedback) => !!feedback.get("learnerId");
-
-const getFeedbacksNotAnswered = (fbs) => fbs.filter(f => !hasLearner(f));
+const getFeedbacksNotAnswered = (fbs) => fbs;
+// TODO: ⬆  How do we know if a student hasn't started yet?
```

Comment:
Right, I had the same problem when I started looking at this. I guess we should check if there's an answer for a given activity. We have this data available. Note that the dashboard has some selectors that calculate progress for each student, so maybe we could reuse it.